Rakefile

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 require 'rubocop/rake_task'
 require 'rake/testtask'
-
 RuboCop::RakeTask.new
 
+require 'scraper_test'
+ScraperTest::RakeTask.new.install_tasks
+
 Rake::TestTask.new do |t|
   t.test_files = FileList['test/**/*_test.rb']
 end
 
-task default: %w(test rubocop)
+task test: 'test:data'
+task default: %w(rubocop test test:data)
```

Comment:
I think you can remove `test:data` from the default task, since it will be run as part of the `test` task (because the previous line declares it as a dependency of the `test` task).